src/Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -129,10 +129,9 @@ public static class WebApiServiceCollectionExtensions
 
                 options.Events.OnTokenValidated = async context =>
                 {
-                    await onTokenValidatedHandler(context).ConfigureAwait(false);                
+                    await onTokenValidatedHandler(context).ConfigureAwait(false);
                     context.HttpContext.StoreTokenUsedToCallWebAPI(context.SecurityToken as JwtSecurityToken);
                     context.Success();
-                    await Task.FromResult(0).ConfigureAwait(false);
```

Comment:
Removed this line which is no longer necessary given we now call `await onTokenValidatedHandler(context).ConfigureAwait(false);`
Thanks @jg11jg for the heads-up in your comments for #154